addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComWindMessage.java

```diff
@@ -86,16 +75,16 @@ public RFXComWindMessage(byte[] data) throws RFXComException {
 
     @Override
     public String toString() {
-        return super.toString()
-            + ", Sub type = " + subType
-            + ", Device Id = " + getDeviceId()
-            + ", Wind direction = " + windDirection
-            + ", Wind gust = " + windSpeed
-            + ", Average wind speed = " + avgWindSpeed
-            + ", Temperature = " + temperature
-            + ", Chill temperature = " + chillTemperature
-            + ", Signal level = " + signalLevel
-            + ", Battery level = " + batteryLevel;
+        return super.toString()
+            + ", Sub type = " + subType
+            + ", Device Id = " + getDeviceId()
+            + ", Wind direction = " + windDirection
+            + ", Wind gust = " + windSpeed
+            + ", Average wind speed = " + avgWindSpeed
+            + ", Temperature = " + temperature
+            + ", Chill temperature = " + chillTemperature
+            + ", Signal level = " + signalLevel
+            + ", Battery level = " + batteryLevel;
```

Comment:
All the messages are using string concatenation instead of `StringBuilders` in their `toString()` methods which is of course not good performance wise.

Reply:
Imho this is only true inside an iteration, in fact Intellij will even suggest to convert the StringBuilder to a String, just like the compiler would do as far as I remember.

![image](https://user-images.githubusercontent.com/2625279/31462112-c5cfd982-aecb-11e7-906a-0e15ff414275.png)